flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/models/NotificationDetails.java

```diff
@@ -36,6 +36,7 @@ public class NotificationDetails implements Serializable {
   private static final String ONGOING = "ongoing";
   private static final String STYLE = "style";
   private static final String ICON = "icon";
+  private static final String SILENT = "silent";
```

Comment:
Nit: you might want to write this below `ONGOING` to keep the order with the rest

Reply:
Fixed :)